content/events/2024-02-26-hackathon-image-analysis-in-galaxy/index.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Hackathon Image Analysis in Galaxy
+date: '2024-02-26'
+days: 5
+tease: "This hackathon addresses the improvement of Galaxy for image analysis"
+continent: EU
+location:
+  name: "University of Freiburg, Germany"
+contact: "Beatriz Serrano-Solano, Yi Sun, Björn Grüning"
+tags: [hackathon, imaging]
+subsites: [all]
+---
+
+# General information
+
+- **Date**: Feb 26th - Mar 3rd 2024 (full week)
+- **Format**: hybrid
+- **Location**: Seminar room at Georges-Köhler-Allee 79, 79104 Freiburg
+- **Time**: full day, 9/10am CEST to open end
+- **Cost**: Free
+- **Contact**: Beatriz Serrano-Solano ([beatriz.serrano.solano@eurobioimaging.eu](mailto:beatriz.serrano.solano@eurobioimaging.eu)), Yi Sun ([yi.sun@embl.de](mailto:yi.sun@embl.de), Björn Grüning ([gruening@informatik.uni-freiburg.de](mailto:gruening@informatik.uni-freiburg.de)
```

Comment:
can we remove this line so that we don't receive much spam? it's already in the form :)